app/views/dishes/show.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1> Dish </h1>
+
+<p><b>Name: </b><%= @dish.name %></p>
+<p><b>Description:</b> <%= @dish.description %></p>
+<p><b>Calorie Count: </b><%= @dish.calorie_count %></p>
+<><b>Chef: </b><%= @dish.chef.name %>
+
+<h3>List of Ingredients:</h3> 
+<% @dish.ingredients.each do |ingredient| %>
+  <%= ingredient.name %> <%= button_to "Delete #{ingredient.name}", "/dishes/#{@dish.id}/ingredients/#{ingredient.id}", method: :delete %>
+<% end %>
+
+<%= form_with url: "/dishes/#{@dish.id}", method: :patch do |f| %>
+  <%= f.text_field :add_ingredient %>
+  <%= f.submit "Submit" %>
+<% end %>
```

Comment:
A couple things about this form, while this form is what I would consider RESTful, it is not RESTful in what it's aiming to accomplish. The task we are given is to associate a new ingredient to a dish, this does not happen on the dish resource, this is a new relationship being CREATED on the DISH_INGREDIENTS table between a dish and an ingredient. We are not updating anything on the dish table itself, therefore a `PATCH` to "/dishes/#{@dish.id}" is incorrect.

Therefore, your form should be making a `POST` request to `"/dishes/#{@dish.id}/ingredients"`.

```erb
<%= form_with url: "/dishes/#{@dish.id}/ingredients", method: :post do |f| %>
  <%= f.text_field :add_ingredient %>
  <%= f.submit "Submit" %>
<% end %>
```

My only other suggestion would be to add a label field so the user knows what they should be doing: `<%= f.label "Add Ingredient" %>`